CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -33,7 +35,7 @@ class EventDetailsVC: UIViewController {
         self.titleLabel.text = self.event?.title
         self.dateLabel.text = self.event?.startDate.toString(dateStyle: .medium, timeStyle: .none).uppercased()
         self.descriptionLabel.text = self.event?.summary
-        self.locationButton.setTitle(self.event?.location?.street, for: .normal)
+        self.locationButton.setTitle(self.locationButtonTitle + "\(self.event?.location?.street)" +  " , \(self.event?.location?.city)", for: .normal)
```

Comment:
This probably shows up as a warning in Xcode. Since self.event? is an optional value, using it in a string will look like "Optional(value)" rather than "value". You should unwrap it and maybe provide a default value by using ??.